### DISCUSSION TITLE
Activating docker-FreePBX with Sangoma

### DISCUSSION THREAD

**Discussion by woobenjamin** [tiredofit/docker-freepbx#204] [Q&A]
Hi all, 

I'm wondering if someone could assist me in activating my install with Sangoma. I want the SysAdmin module, and I've bought the 25 year license, but I don't know how to activate the install so I can load the license. Thank you.

b

**Comment by gillian2020** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
